common/beerocks/bcl/include/bcl/son/son_wireless_utils.h

```diff
@@ -69,6 +69,11 @@ inline bool operator<(const sPreference &rhs, const sPreference &lhs)
     return rhs.oper_class < lhs.oper_class || rhs.preference < lhs.preference ||
            rhs.reason < lhs.reason;
 }
+
+inline bool operator==(const sWifiChannel &wifi_channel, const uint8_t channel)
+{
+    return wifi_channel.channel == channel;
+}
```

Comment:
As noted somewhere else, this is only needed for the std::find calls, and I think we can avoid them.

Reply:
Done.